hadoop-ozone/dist/src/main/smoketest/freon/read-write-key.robot

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Test freon read/write key commands
+Resource            ../ozone-lib/freon.robot
+Test Timeout        5 minutes
+
+*** Variables ***
+${PREFIX}    ${EMPTY}
+
+*** Test Cases ***
+Pre-generate 100 keys of size 1 byte each to Ozone
+    ${result} =        Execute          ozone freon ork -n 1 -t 10 -v voltest -b buckettest -p performanceTest
+
+Read 10 keys from pre-generated keys
+    ${result} =        Execute          ozone freon ockrw -n 10 -t 10 -r 100 -v voltest -b buckettest -p performanceTest
+
+Read 10 keys' metadata from pre-generated keys
+    ${result} =        Execute          ozone freon ockrw -n 10 -t 10 -m -r 100 -v voltest -b buckettest -p performanceTest
+
+Write 10 keys of size 1 byte each from key index 0 to 99
+    ${result} =        Execute          ozone freon ockrw -n 10 -t 10 --percentage-read -r 100 -v voltest -b buckettest -p performanceTest
```

Comment:
Add a test that lists keys under the prefix and checks if the result matches the expected count and if the object size is the expected size.

Reply:
Sure! added.